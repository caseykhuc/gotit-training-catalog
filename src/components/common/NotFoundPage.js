import React from 'react';

const NotFoundPage = () => (
  <div className="d-flex align-items-center justify-content-center" style={{ height: '20rem' }}>
    <div>
      <h2>404 Not Found</h2>
      <div className="error-details">
            Requested page not found!
      </div>
    </div>
  </div>
)

export default NotFoundPage;
