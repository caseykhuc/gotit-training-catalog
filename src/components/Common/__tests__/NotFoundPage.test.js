import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../NotFoundPage';

describe('component/Common/NotFoundPage', () => {
  let props;
  let wrapper;

  const update = () => {
    wrapper.update();
  };
  const setup = () => {
    wrapper = shallow(<NotFoundPage {...props} />);
    update();
  };
  beforeEach(() => {
    props = {
    }
  });
  it('should render correctly', () => {
    setup();
    expect(wrapper).toMatchSnapshot();
  });
})
