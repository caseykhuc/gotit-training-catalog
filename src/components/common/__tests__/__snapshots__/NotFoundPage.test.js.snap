// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component/NotFoundPage should render correctly 1`] = `ShallowWrapper {}`;
